PopcornEditor.js

```diff
@@ -10,9 +10,30 @@ function PopcornEditor() {
 
   this.init = function (el, url) {
     var editor = (typeof el === 'string') ? document.getElementById(el) : el,
-        url = url || 'PopcornEditor/editor.html';
+        url = url || 'PopcornEditor/editor.html',
+        loadingDiv = null,
+        loadingSpinner = null;
 
     window.addEventListener('message', onmessage);
+    this.listen('loaded', function () {
+      document.querySelector('#loading').remove();
+    });
+
+    loadingDiv = document.createElement('div');
+    loadingDiv.setAttribute('id', 'loading');
+    loadingDiv.style.width = '100%';
+    loadingDiv.style.height = '100%';
+    loadingDiv.style.backgroundColor = 'white';
+    loadingDiv.style.display = 'flex';
+    loadingDiv.style.justifyContent = 'center';
```

Comment:
If it meshes with the architecture you're thinking here, I'd put this in `index.html` or some css file, just to be clean.